hive-metastore/src/test/java/org/apache/iceberg/hive/HiveCreateReplaceTableTest.java

```diff
@@ -292,7 +293,11 @@ public void testCreateTableTxnWithGlobalTableLocation() {
 
     Transaction txn =
         catalog.newCreateTableTransaction(
-            TABLE_IDENTIFIER, SCHEMA, SPEC, "file:///" + tableLocation, Maps.newHashMap());
```

Comment:
are you sure about indentation? Probably iceberg has some maven/gradle goal to reformat as expected

Reply:
I'm sure I have configured google java format